src/EditorFeatures/VisualBasicTest/CodeActions/ConvertToInterpolatedString/ConvertToInterpolatedStringTests.vb

```diff
@@ -418,4 +418,16 @@ Module T
 End Module</File>.ConvertTestSourceTag()
         Await TestMissingAsync(text)
     End Function
+
+    <WorkItem(13605, "https://github.com/dotnet/roslyn/issues/13605")>
+    <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertToInterpolatedString)>
+    Public Async Function TestInvocationWithNullArguments() As Task
+        Dim text = <File>
+Module Module1
+    Sub Main()
+        [|TaskAwaiter|]
+    End Sub
+End Module</File>.ConvertTestSourceTag()
```

Comment:
note: you can just use a VB multiline string literal now.  That's the preferred style in Vb tests now.